static/js/hotkey.js

```diff
@@ -194,6 +195,9 @@ function process_hotkey(e) {
 
     // Process hotkeys specially when in an input, select, textarea, or send button
     if ($('input:focus,select:focus,textarea:focus,#compose-send-button:focus').length > 0) {
+        if (event_name === 'left_arrow') {
+            return false;
+        }
```

Comment:
I think we probably want the left-arrow hotkey to work in the event that the compose box is open, but empty; so probably makes sense to use code similar to that for the `up_arrow` code path a little below here to decide whether to return.